tasks/aiida-deps-debian.yml

```diff
@@ -20,5 +25,30 @@
       - python3-psycopg2
       - python3-gi
       - python3-gi-cairo
+      - python3-venv
       - gir1.2-gtk-3.0
       - python3-tk
+
+- name: Fetch location of the Postgresql binaries
+  command: find /usr/lib/postgresql/ -name bin
```

Comment:
Just for context, [pgtest](https://github.com/jamesnunn/pgtest/blob/master/pgtest/pgtest.py#L100) in the end does the same, so I guess there really is no better way.

Reply:
I do not know. Possibly not.